application/auth/helpers.py

```diff
@@ -28,31 +28,29 @@
 SERVER_ERROR_MSG = 'Authorization failed. Please contact support.'
 
 
-def save_user():
-    user = User.query.filter_by(email=session.get("email")).first()
+def save_user(payload):
+    user = User.query.filter_by(email=payload.get("email")).first()
```

Comment:
Aren't you supposed to validate if email exists?

Reply:
you mean in the payload? There's no need for that validation as we are using google main, by default that payload contains the user's email